MERN/Topic/Frontend/React/1.2_Getting_started_with_React.md

```diff
@@ -1,70 +1,81 @@
-    <br>
+   Download the Git CLI from https://git-scm.com/downloads and then install the **Git CLI** by opening the downloaded installer.
+
+   > Test that the **Git CLI** was installed successfully by running the command `git --version`.  
+   > <br>
 
 3. ### **A Code Editor :**
-    A recommended code editor would be [Visual Studio Code](https://code.visualstudio.com/) .  
-    You can also go for  
-    - [Sublime Text](https://www.sublimetext.com/)
-    - [ATOM](https://atom.io/)
-    - [Brackets](https://brackets.io/)
-    - [Notepad++](https://notepad-plus-plus.org/)
-    <br>
+   A recommended code editor would be [Visual Studio Code](https://code.visualstudio.com/) .  
+   You can also go for
+   - [Sublime Text](https://www.sublimetext.com/)
+   - [ATOM](https://atom.io/)
+   - [Brackets](https://brackets.io/)
+   - [Notepad++](https://notepad-plus-plus.org/)
+     <br>
 
 ## **How to Create React App**
+
 <br>
 
-**React** is a free and open-source front-end JavaScript library for building user interfaces or UI components.  
+**React** is a free and open-source front-end JavaScript library for building user interfaces or UI components.
+
 - Now to create a **React App**, we first need to open a terminal in VS Code(or any editor you are using) then :
-    - We can use the tool **npx** (if we want some packages to be included in our application, but we don’t want to download it then we can use npx ). npx gives us the ability to use the `create-react-app` package without having to first install it on our computer.
-      
-       Command to create React App using npx :  
-        ```
-        npx create-react-app my-react-app
-        ```  
-    - Command to create React App using npm :
-      ```
-      npm init react-app my-react-app
-      ``` 
-      *Above command is available in `npm` 6+.*  
-
-    - Command to create React App using yarn :
-      ```
-      yarn create react-app my-react-app
-      ```
-      *Above command is available in `Yarn` 0.25+.*  
-      <br>
-- Once we have executed the above command, a folder named **"my-react-app"** will be created where we specified on our computer and all of the packages it requires will be automatically installed.  
-
-- Now after creating the `React App` to test whether our app is working correctly or not, open **"my-react-app"** folder with VS Code then in VS Code go to view and open up your terminal then write the following command :  
+
+  - We can use the tool **npx** (if we want some packages to be included in our application, but we don’t want to download it then we can use npx ). npx gives us the ability to use the `create-react-app` package without having to first install it on our computer.
+
+    Command to create React App using npx :
+
     ```
-    npm start
-    ```  
-    or  
+    npx create-react-app my-react-app
     ```
-    yarn start
+
+  - Command to create React App using npm :
+
+    ```
+    npm init react-app my-react-app
```

Comment:
Great work. Some minor changes are suggested, Whenever writing such type of commands instead of using hardcoded values, try to show it like this:
General syntax:
`npm init react-app <PROJECT_NAME>` 
For creating a project with a name of my-react-app, it would become:
`npm init react-app my-react-app`

Reply:
done !